src/odemis/acq/stream/_sync.py

```diff
@@ -167,6 +167,7 @@ def __init__(self, name, streams):
         self._acq_done = threading.Event()
 
         # Start threading event for live update overlay
+        self._current_scan_area = (0, 0, 0, 0)  # currently scanned area location based on px_idx
```

Comment:
Would be nice to add here what happens if current_scan_area is defined as None (no live update will be given because of the if statement in _updateImage).

Reply:
Updated the comment.